packages/medusa/src/api-v2/admin/price-lists/[id]/prices/route.ts

```diff
@@ -0,0 +1,68 @@
+import {
+  removePriceListPricesWorkflow,
+  upsertPriceListPricesWorkflow,
+} from "@medusajs/core-flows"
+import {
+  AuthenticatedMedusaRequest,
+  MedusaResponse,
+} from "../../../../../types/routing"
+import { listPriceLists } from "../../queries"
+import {
+  adminPriceListRemoteQueryFields,
+  defaultAdminPriceListFields,
+} from "../../query-config"
+import {
+  AdminDeletePriceListsPriceListPricesReq,
+  AdminPostPriceListsPriceListPricesReq,
+} from "../../validators"
+
+export const POST = async (
+  req: AuthenticatedMedusaRequest<AdminPostPriceListsPriceListPricesReq>,
+  res: MedusaResponse
+) => {
+  const { prices } = req.validatedBody
+  const id = req.params.id
+  const workflow = upsertPriceListPricesWorkflow(req.scope)
+  const { errors } = await workflow.run({
+    input: {
+      price_lists_data: [{ id, prices }],
+    },
+    throwOnError: false,
+  })
+
+  if (Array.isArray(errors) && errors[0]) {
+    throw errors[0].error
+  }
+
+  const [[priceList]] = await listPriceLists({
+    container: req.scope,
+    remoteQueryFields: adminPriceListRemoteQueryFields,
+    apiFields: defaultAdminPriceListFields,
+    variables: { filters: { id }, skip: 0, take: 1 },
+  })
+
+  res.status(200).json({ price_list: priceList })
+}
+
+export const DELETE = async (
```

Comment:
suggestion: Should we move this to `POST /admin/price-lists/:id/prices/remove`?

Reply:
Yes, it's already part of the work I'm doing next along with update prices.